### ISSUE TITLE
The `dollar-variable-colon-newline-after` rule should ignore maps

### ISSUE THREAD

**Issue by sindresorhus** [kristerkari/stylelint-scss#203]
Or at least have an option to ignore them.

This currently causes a violation:

```scss
$map: (
	'foo': 'foo',
	'bar': 'bar'
)
```

But it would look weird to write it as:

```scss
$map:
	(
		'foo': 'foo',
		'bar': 'bar'
	);
```

---

With stylelint-scss 2.3.0

**Comment by kristerkari** [comment]
This the same issue as the one reported here, right?
https://github.com/kristerkari/stylelint-scss/issues/172

**Comment by kristerkari** [comment]
I guess that we could just ignore maps by default as it looks weird to write them with a newline.

**Comment by kristerkari** [comment]
I added a comment to ask for feedback to decide what the default behavior for `dollar-variable-colon-newline-after`rule should be:
https://github.com/kristerkari/stylelint-scss/issues/172#issuecomment-363594714
